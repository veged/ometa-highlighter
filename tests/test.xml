<?xml version="1.0" encoding="utf-8"?>
<root>
    <grass type="nice"/>

    <tree>
        <leaf id="1"/>
        <branch>
            <leaf id="2"/>
            <leaf id="3"/>
        </branch>
    </tree>
</root>
