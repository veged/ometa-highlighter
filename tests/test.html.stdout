<pre class="ohl ohl-xml"><code><span class="ohl-comment ohl-comment-preproc"><span class="ohl-punctuation">&lt;?</span>xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;<span class="ohl-punctuation">?&gt;</span></span>
<span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>root<span class="ohl-punctuation">&gt;</span></span>
    <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>grass <span class="ohl-name ohl-name-attribute">type<span class="ohl-punctuation">=</span><span class="ohl-string ohl-string-double"><span class="ohl-punctuation">&quot;</span>nice<span class="ohl-punctuation">&quot;</span></span></span><span class="ohl-punctuation">/&gt;</span></span>

    <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>tree<span class="ohl-punctuation">&gt;</span></span>
        <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>leaf <span class="ohl-name ohl-name-attribute">id<span class="ohl-punctuation">=</span><span class="ohl-string ohl-string-double"><span class="ohl-punctuation">&quot;</span>1<span class="ohl-punctuation">&quot;</span></span></span><span class="ohl-punctuation">/&gt;</span></span>
        <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>branch<span class="ohl-punctuation">&gt;</span></span>
            <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>leaf <span class="ohl-name ohl-name-attribute">id<span class="ohl-punctuation">=</span><span class="ohl-string ohl-string-double"><span class="ohl-punctuation">&quot;</span>2<span class="ohl-punctuation">&quot;</span></span></span><span class="ohl-punctuation">/&gt;</span></span>
            <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;</span>leaf <span class="ohl-name ohl-name-attribute">id<span class="ohl-punctuation">=</span><span class="ohl-string ohl-string-double"><span class="ohl-punctuation">&quot;</span>3<span class="ohl-punctuation">&quot;</span></span></span><span class="ohl-punctuation">/&gt;</span></span>
        <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;/</span>branch<span class="ohl-punctuation">&gt;</span></span>
    <span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;/</span>tree<span class="ohl-punctuation">&gt;</span></span>
<span class="ohl-name ohl-name-tag"><span class="ohl-punctuation">&lt;/</span>root<span class="ohl-punctuation">&gt;</span></span>
</code></pre>